diagrams/c4/2_container.puml

```diff
@@ -0,0 +1,45 @@
+﻿@startuml
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+
+Person(roleUser, "User")
+Person(roleAdmin, "Administrator")
+
+System_Ext(sysBitcoin, "Bitcoin Network")
+System_Ext(sysEthereum, "Ethereum Network")
+System_Ext(sysEmail, "Email Delivery")
+System_Ext(sysUtilityProvider, "Utility Provider")
+
+System_Boundary(sysCryptobank, "Cryptobank") {
```

Comment:
Which container will manage operations with virtual cards?

Reply:
I'm going to implement card functionality in `Account Service`.